rockcraft.yaml

```diff
@@ -118,6 +129,10 @@ parts:
       - bin/patchelf
 
   chisel:
+    # This must be after the 'snapcraft' part. Otherwise, `usr/lib/python3.12/sitecustomize.py`
+    # created by the Python plugin from the 'snapcraft' part never gets staged.
```

Comment:
This is a naive comment because I don't understand why this is happening or if there is a better fix.